### PR TITLE
[13_0_X] Addition of 2023 WFs and Fixing NANO step (Fix typo)

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
This is a clone of https://github.com/cms-sw/cmssw/pull/42089 with typo-fix, MINIAOOD => MiniAOD [[here](https://github.com/cms-sw/cmssw/pull/42089#discussion_r1253016974)]

#### PR validation:
Run wf `140.001` which failed in IB before.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:
This is a backport PR.

### CODE REVIEW

**Comment by srimanob** [comment]
test parameters:
- workflows = 140.001,140.002,140.004,140.005,140.115,140.116
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

Change is expected due to HLT step, as mentioned in https://github.com/cms-sw/cmssw/pull/42089

**Comment by sunilUIET** [comment]
+pdmv

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -443,87 +443,111 @@
...
+workflows[140.109] = ['',['RunEGamma2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMEGAMMA0RUN3_reHLT']]
+workflows[140.110] = ['',['RunTau2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMTAURUN3_reHLT']]
+workflows[140.111] = ['',['RunMuon2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMMUON0RUN3_reHLT']]
+workflows[140.112] = ['',['RunMuonEG2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMMUONEGRUN3_reHLT']]
+workflows[140.113] = ['',['RunNoBPTX2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMNOBPTXRUN3_reHLT']]
+workflows[140.114] = ['',['RunHcalNZS2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMHCALNZSRUN3_reHLT']]
+workflows[140.115] = ['',['RunHLTPhysics2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMHLTPHYSICSRUN3_reHLT']]
+workflows[140.116] = ['',['RunCommissioning2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMCOMMISSIONINGRUN3_reHLT']]
+workflows[140.117] = ['',['RunCosmics2022D','HLTDR3_2022','RECOCOSMRUN3_reHLT','SKIMCOSMICSRUN3_reHLT']]
 #workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2023','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
```
Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_standard.py:525]:
```suggestion
#workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2022','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
```
can avoid problems if and when uncommenting this line

**Review by perrotta — COMMENTED** [review]
Since this already missed the deadline for the CMSSW_13_0_10 release, I would profit and fix also the comment line, in order to avoid possible problems when it will get uncommented.

**Comment by perrotta** [comment]
backport of #42069
(Overall:
- partial backport of https://github.com/cms-sw/cmssw/pull/42069
- partial backport of https://github.com/cms-sw/cmssw/pull/41629
- backport of the bug fixes in https://github.com/cms-sw/cmssw/pull/42047))

**Comment by srimanob** [comment]
@cmsbuild please test

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 2]
```diff
@@ -443,86 +443,110 @@
...
-workflows[140.110] = ['',['RunTau2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMTAURUN3_reHLT']]
-workflows[140.111] = ['',['RunMuon2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUON0RUN3_reHLT']]
-workflows[140.112] = ['',['RunMuonEG2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUONEGRUN3_reHLT']]
-workflows[140.113] = ['',['RunNoBPTX2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMNOBPTXRUN3_reHLT']]
-workflows[140.114] = ['',['RunHcalNZS2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHCALNZSRUN3_reHLT']]
-workflows[140.115] = ['',['RunHLTPhysics2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHLTPHYSICSRUN3_reHLT']]
-workflows[140.116] = ['',['RunCommissioning2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMCOMMISSIONINGRUN3_reHLT']]
-workflows[140.117] = ['',['RunCosmics2022D','HLTDR3_2023','RECOCOSMRUN3_reHLT','SKIMCOSMICSRUN3_reHLT']]
-#workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2023','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
+workflows[140.104] = ['',['RunZeroBias2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMZEROBIASRUN3_reHLT']]
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_standard.py:512]: we need a ZB flavor

Comment (slava77) [reply Configuration/PyReleaseValidation/python/relval_standard.py:512]: > we need a ZB flavor

ah, there is  `140.063] = ['',['RunZeroBias2022D'`.
Is this just a no-DQM production like setup?

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_standard.py:512]: Good question @slava77 

Looking in [master](https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_standard.py#L497C1-L497C1), this workflow produce DQM from [RECONANORUN3_ZB_reHLT_2022](https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2910) (or RECONANORUN3_reHLT in 13_0). But there is no harvesting step.

@cms-sw/pdmv-l2 Is this your intention, or missing?

Comment (sunilUIET) [reply Configuration/PyReleaseValidation/python/relval_standard.py:512]: > master

@srimanob @slava77 I am not sure of the use case of WFs 140.104-140.118. For consistency, I changed the DQM sequence to ZeroBias 140.104 as well but missed the changes in the Harvesting step. As the ZeroBias DQM request is initially from tracking @mmusich, as per their need either we can revert back to standard DQM or update harvesting step for this WF.

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

I would propose to merge this PR as is, and fix for harvesting in different PRs, follow the fix in master.

**Comment by sunilUIET** [comment]
+pdmv

**Comment by perrotta** [comment]
+1

**Pull request merged**